### ISSUE TITLE
Not Accessible

### ISSUE THREAD

**Issue by robertmausser** [cgkineo/adapt-hint#4]
Any desire or interest to make this accessible? 

Thanks!

**Comment by moloko** [comment]
@robertmausser I think it might be more that the README is out of date. I'm certainly seeing various references to accessibility in the code.

**Comment by moloko** [comment]
@robertmausser regardless, the new v5 compatible version should have good a11y support, please see https://github.com/cgkineo/adapt-hint/pull/5
